src/Exception/ArrayValueCalculator.java

```diff
@@ -0,0 +1,21 @@
+package Exception;
+
+public class ArrayValueCalculator {
+    public static int doCalc(String[][] inputArray) throws ArrayDataException, ArraySizeException {
+        if (inputArray.length != 4 || inputArray[0].length != 4) {
```

Comment:
array in java is can be not simetrical inputArray[0].length  != inputArray[1].length, also  inputArray is a bad var name